tests/Integrations/Greenhouse/GreenhouseTest.php

```diff
@@ -0,0 +1,501 @@
+<?php
+
+/**
+ * Tests for Greenhouse integration file.
+ */
+
+namespace EightshiftFormsTests\Integrations\Greenhouse;
+
+use Brain\Monkey\Functions;
+use EightshiftForms\Hooks\Filters;
+use EightshiftFormsTests\BaseTest;
+use EightshiftFormsTests\Mocks\MockGreenhouse;
+class GreenhouseTest extends BaseTest
+{
+	protected function _inject(DataProvider $dataProvider)
+	{
+		$this->dataProvider = $dataProvider;
+	}
+
+	protected function _before()
+	{
+		parent::_before();
+		$this->greenhouse = $this->diContainer->get(MockGreenhouse::class);
+	}
```

Comment:
I think you should add `_after()` to cleanup your test.

Reply:
what should I add in after?

Reply:
I think `parent::_after();`. And before that you can just set the greenhouse property to null.

Reply:
Parent class (`BaseTest`) already does `_after()` like so:

```php
	protected function _after()
	{
		Monkey\tearDown();
	}
```

No need to add `_after()` in your test unless you need to tearDown something else as well. CC @dingo-d @iruzevic